src/braket_container.py

```diff
@@ -254,41 +225,74 @@ def install_additional_requirements() -> None:
         log_failure_and_exit(f"Unable to install requirements.\nException: {e}")
 
 
-def run_customer_code_as_process(entry_point: str) -> int:
+def extract_customer_method(entry_point: str) -> Callable:
+    """
+    Converts entry point to a runnable function.
     """
-    When provided the name of the package and the method to run, we run them as a process.
+    if entry_point.find(":") >= 0:
+        str_module, _, str_method = entry_point.partition(":")
+        customer_module = importlib.import_module(str_module)
+        customer_method = getattr(customer_module, str_method)
+    else:
+        def customer_method():
+            return runpy.run_module(entry_point, run_name="__main__")
+    return customer_method
+
+
+def wrap_customer_method(customer_method: Callable) -> Callable:
+    def wrapped_customer_method(**kwargs):
+        try:
+            return customer_method(**kwargs)
+        except Exception as e:
+            exception_type = type(e).__name__
+            exception_string = (
+                exception_type
```

Comment:
What happens if we always just print what's in the else statement?

Will this handle customer's custom exceptions properly (ie. if they don't have any args, but have an additional error message)?

Reply:
Some exceptions don't have args, ie an AssertionError. We would like to print this out as `AssertionError` and not `AssertionError: `

I believe this should be pretty flexible for custom exception types